airflow/providers/amazon/aws/operators/rds.py

```diff
@@ -551,6 +551,92 @@ def execute(self, context: 'Context') -> str:
         return json.dumps(delete_subscription, default=str)
 
 
+class RdsCreateDbInstanceOperator(RdsBaseOperator):
+    """
+    Creates an RDS DB instance
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:RdsCreateDbInstanceOperator`
+
+    :param db_instance_identifier: The DB instance identifier, must start with a letter and
+        contain from 1 to 63 letters, numbers, or hyphens
+    :param db_instance_class: The compute and memory capacity of the DB instance, for example db.m5.large
+    :param engine: The name of the database engine to be used for this instance
+    :rds_kwargs: Named arguments to pass to boto3 RDS client function ``create_db_instance``
+    """
+
+    def __init__(
+        self,
+        *,
+        db_instance_identifier: str,
+        db_instance_class: str,
+        engine: str,
+        rds_kwargs: Optional[Dict] = None,
+        aws_conn_id: str = "aws_default",
```

Comment:
Would you mind adding `aws_conn_id` to the docstring of this operator and `RdsDeleteDbInstanceOperator`? I think it would be nice for users to explicitly see this in the Python API documentation for these operators.

Reply:
Makes sense, added :)